client/components/ChartBox.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { useState } from 'react';
+import {
+    Chart as ChartJS,
+    LinearScale,
+    CategoryScale,
+    BarElement,
+    PointElement,
+    LineElement,
+    Legend,
+    Tooltip,
+} from 'chart.js';
+import { Chart, Line } from 'react-chartjs-2';
+import { SelectedProject } from './Interfaces';
+
+ChartJS.register(
+    LinearScale,
+    CategoryScale,
+    BarElement,
+    PointElement,
+    LineElement,
+    Legend,
+    Tooltip
+);
+const labels = ['Query1', 'Query2', 'Query3', 'Query4', 'Query5', 'Query6', 'Query7'];
+
+export const options = {
+    responsive: true,
+    plugins: {
+        legend: {
+            position: 'top' as const,
+        },
+        title: {
+            display: true,
+            text: 'Chart.js Line Chart',
+        },
+    },
+};
+export interface ISState {
+    style: {
+        chartOne: string;
+        chartTwo: string;
+        chartThree: string;
+        chartFour: string;
+    };
+}
+
+export interface IProps {
+    project: SelectedProject['project'];
+}
+
+// eslint-disable-next-line react/function-component-definition
+const ChartBox: React.FC<SelectedProject> = ({ project }) => {
+    const timeFromProjectProps: number[] = [];
+    const depthFromProjectProps: number[] = [];
+    const complexityFromProjectProps: number[] = [];
+
+    // eslint-disable-next-line array-callback-return
+    project?.queries.map((query) => {
```

Comment:
This loop on line 58 can use a `foreach` since you aren't returning anything. It will also solve your eslint error